tests/clock/ClockUtils.py

```diff
@@ -0,0 +1,384 @@
+import logging
+import random
+import datetime
+import allure
+import time
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.clock.ClockConsts import ClockConsts
+from tests.common.errors import RunAnsibleModuleFail
+
+
+class ClockUtils:
+    @staticmethod
+    def run_cmd(duthosts, cmd, param=''):
+        """
+        @summary:
+            Run a given command and return its output.
+            * A successful command returns an empty output (''), while failure returns an error message
```

Comment:
I believe this needs to be rephrased as the output will not be empty for show commands